contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -455,6 +449,26 @@ def get_title(item):
     return item["title"] if item["parent_id"] else item["original_channel_name"]
 
 
+def consolidate_extra_fields(item):
+    extra_fields = item.get("extra_fields")
+    if item["kind"] == content_kinds.EXERCISE:
+        m = extra_fields.get("options", {}).pop("m", None)
+        n = extra_fields.get("options", {}).pop("n", None)
+        mastery_model = extra_fields.get("options", {}).pop("mastery_model", None)
+        if not extra_fields.get("options", {}).get("completion_criteria", {}):
```

Comment:
Would be good to add an extra condition here checking `mastery_model is not None` - and add an extra test for this condition - basically, when the exercise hasn't had a mastery model set on it.